### DISCUSSION TITLE
[Feature Request : Taskbar] add option to combine taskbar buttons and hide labels

### DISCUSSION THREAD

**Discussion by CorruptionVR** [DustinBrett/daedalOS#303] [Feature Requests]
I would like the option to combine taskbar buttons and hide labels. this would make the webOS more like windows 10.

**Comment by DustinBrett** [comment]
Thanks for the suggestion. Indeed that would be more like Windows 10 and I will add this to feature requests. The main reason I didn't do this at the moment is that the way it is reflects my personal preferences for the settings, and how I liked it back in the Windows 2000 days.

**Comment by JeremyLyokoPPG2020** [comment]
Large Taskbar and combine taskbar buttons + hide labels
<img width="478" height="60" alt="image" src="https://github.com/user-attachments/assets/f4d468d9-45cc-4df9-815f-1aab376ee77e" />

Small Taskbar and combine taskbar buttons + hide labels
<img width="480" height="35" alt="image" src="https://github.com/user-attachments/assets/c303bac2-e96c-40d1-bfd6-cc71edaf4149" />
